content/applications/sales/crm/pipeline/lost_opportunities.rst

```diff
@@ -3,51 +3,51 @@ Manage lost opportunities
 =========================
 
 Not every opportunity results in a successful sale. To keep the pipeline up-to-date, *lost*
-opportunities need to be identified. Specifying why an opportunity was lost provides additional
-insight that can prove useful for future opportunities.
+opportunities need to be identified. Specifying why an opportunity was lost helps identify recurring
+issues, reveal coaching opportunities, and can assist with improving overall sales strategy.
+
+.. note::
+   :doc:`Merging lost opportunities <merge_similar>` with active ones will pull them back into the
+    pipeline.
 
 Mark an opportunity as lost
 ===========================
 
-To mark an opportunity as lost, first open the :menuselection:`CRM app`, and select an opportunity
-from the pipeline, by clicking on its corresponding Kanban card. Doing so reveals that opportunity's
+To mark an opportunity as lost, first open the :menuselection:`CRM` app, then select an opportunity
+from the pipeline by selecting its corresponding Kanban card. This reveals that opportunity's
```

Comment:
Note:
There was nothing wrong with the prior version
We do not need to make superficial wording edits like this.